api/managementpb/backup/backups.proto

```diff
@@ -0,0 +1,66 @@
+syntax = "proto3";
+
+package backup.v1beta1;
+
+option go_package = "api/managementpb/backup;backupv1beta1";
+
+import "google/api/annotations.proto";
+import "google/protobuf/timestamp.proto";
+
+// Status shows current status of Backup.
+enum Status {
+  STATUS_INVALID = 0;
+  PENDING = 1;
+  IN_PROGRESS = 2;
+  PAUSED = 3;
+  SUCCESS = 4;
+  ERROR = 5;
+}
+
+// DataModel is a model used for performing a backup.
+enum DataModel {
+  DATA_MODEL_INVALID = 0;
+  PHYSICAL = 1;
+  LOGICAL = 2;
+}
+
+// Backup represents single Backup Location.
+message Backup {
+  // Machine-readable backup ID.
+  string backup_id = 1;
+  // Backup name
+  string name = 2;
+  // Database vendor e.g. PostgreSQL, MongoDB, MySQL.
+  string vendor = 3;
+  // Machine-readable location ID.
+  string location_id = 4;
+  // Backup location name.
+  string location_name = 5;
+  // Machine-readable backup service ID.
+  string service_id = 6;
+  // Backup service name.
+  string service_name = 7;
+  // Backup data model.
+  DataModel data_model = 8;
+  // Backup status.
+  Status status = 9;
+  // Backup creation time.
+  google.protobuf.Timestamp created_at = 10;
+}
+
+message ListBackupsRequest {}
+
+message ListBackupsResponse {
+  repeated Backup backups = 1;
+}
+
+// Backups service provides public methods for managing Backups.
+service Backups {
+  // ListBackups returns a list of all backups.
+  rpc ListBackups(ListBackupsRequest) returns (ListBackupsResponse) {
+    option (google.api.http) = {
+      post: "/v1/management/backup/Backups/List"
```

Comment:
Any reason why the signature is different from what I see on other /management endpoint?
https://pmmdemo.percona.com/swagger

Reply:
If we follow the same approach as here https://pmmdemo.percona.com/swagger#/SecurityChecks/ListSecurityChecks, then it should probably be `/v1/management/Backups/List`

Reply:
inspired by https://github.com/percona/pmm/blob/release/2.16/api/managementpb/ia/channels.proto#L176

Reply:
Ok, I see your point! But then it should probably mean "inventory", like this `post: "/v1/management/backup/Inventory/List"`. Pls check p.3.1.1 in https://confluence.percona.com/display/PMM/Backup+Management. And also the design below, to stay consistent.
![BackupManagement - Inventory](https://user-images.githubusercontent.com/81549/111313358-5034dc00-8671-11eb-80a6-8df45b3c9050.png)

Reply:
as per discussion renamed to `v1/management/backup/Artifacts/List`